transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventArray.java

```diff
@@ -38,7 +39,8 @@
  * We use {@code fd} if the {@code epoll_data union} to store the actual file descriptor of an
  * {@link AbstractEpollChannel} and so be able to map it later.
  */
-final class EpollEventArray {
+@UnstableApi
+public final class EpollEventArray {
```

Comment:
how is this exposed as public ?

Reply:
Via `io.netty.channel.epoll.Native` which is `public` and has `epollBusyWait` and `epollWait` methods that are also `public` and take an `EpollEventArray` as an argument.